download_action.go

```diff
@@ -128,8 +174,19 @@ func (d *DownloadAction) Run(context *DebosContext) error {
 		return fmt.Errorf("Unsupported URL is provided: '%s'", url.String())
 	}
 
+	// Unpack and Install are mutually exclusive
 	if d.Unpack == true {
 		return UnpackTarArchive(filename, context.rootdir, d.Compression)
+	} else if d.Install == true {
```

Comment:
This should be a seperate action rather then installing directly from a download action, see also the preivous comment :). That way we can also nicely support bundling a .deb in a recipe and install that.